certified-connectors/Flotiq/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "API Key for this API",
+          "tooltip": "Please enter your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#16A085",
+    "capabilities": [],
+    "stackOwner": "Flotiq",
+    "publisher": "CodeWave LLC"
```

Comment:
@andrzejwp Why `publisher` is different from `stackOwner` ?

Reply:
CodeWave is the tech team working on Flotiq - the product, which is a separate entity.